src/components/idleUsers/section/index.tsx

```diff
@@ -1,21 +1,15 @@
 import { FC } from 'react';
-import Card from '@/components/idleMembers/card';
-import classNames from '@/components/idleMembers/section/section.module.scss';
+import Card from '@/components/idleUsers/card';
+import classNames from '@/components/idleUsers/section/section.module.scss';
+import { idleUserArray, idleUser } from '@/interfaces/idleUser.type';
 
-type Props = {
-  heading: string
-  content: Array<string>
-  error: string | null
-  isLoading: boolean
-}
-
-function renderCards(content: Array<string>) {
-  return content.map((idleMember) => (
-    <Card idleMemberUserName={idleMember} key={idleMember} />
+function renderCards(content: idleUser[]) {
+  return content.map((idleUser) => (
+    <Card user={idleUser} key={idleUser.id} />
   ));
 }
 
-const Section: FC<Props> = ({
+const Section: FC<idleUserArray> = ({
```

Comment:
`idleUserArray` might not exit after PR #378 will be merged. 
[Link](https://github.com/Real-Dev-Squad/website-status/pull/378/files#diff-46824615da47fe1362ea6861d10da582e83535582d85047f1a814b6da4f3037b)